### ISSUE TITLE
Crashing when right clicking monitor to camera for Securitycraft

### ISSUE THREAD

**Issue by Gamerbolts** [XCompWiz/LookingGlass#28]
I am not sure what is causing this. But I have a big modpack. So needed NotEnoughIDs. Things were working fine, added Lookingglass to get the extra features. 

Whenever I right clicked to connect the camera it crashed my client. Took out every mod except Securitycraft, Lookingglass, and NotenoughIDs. 

Tried right clicking and was met with this http://imgur.com/a/sYoQO. Can't move can't do anything. Just sitting there. This is the log. http://pastebin.com/S83DYPxD 

I apologize that it is not complete. Pastebin didn't let me paste the full one. 

I have posted this on Securitycraft, Lookingglass and NotEnoughIDs githubs. 

Anyone please help!

Thank you.

**Comment by XCompWiz** [comment]
Pretty sure this is SecurityCraft having some issue with the latest version of LookingGlass. You should be able to go to a slightly earlier version of LookingGlass. SecurityCraft is no longer updating for that version of Minecraft to my knowledge, so that's the only fix.

**Comment by Gamerbolts** [comment]
Thanks for the reply. I went to 0.2.0.00 and it was the same issue. Just those 3 mods. Then I went to 0.1.2.00 and when I right clicked it says shutting down internal server. And yeah Securitycraft isn't updating anymore unfortunately. Thing is it works by themselves just those two mods. but adding in NotEnoughIDs is where it stops working, I just didn't know if something on your end could fix what happened or not. Since the author of NotEnoughIDs always claims it's not his mod that causes the issue.

**Comment by XCompWiz** [comment]
I'm sorry. I guessed based on what you said without looking through the log. Looks like a packet error somewhere.
I don't know what's causing it, but I'll look into this issue more when I next have time to work on LookingGlass. I'll see what I can do. :)

**Comment by Gamerbolts** [comment]
ok awesome, thank you :)

**Comment by Gamerbolts** [comment]
Just curious if there is any update on this? Would love to use LookingGlass in my modpack but still crashes with NotEnoughIDs

**Comment by XCompWiz** [comment]
I've not looked at LookingGlass yet. Sorry. :)

**Comment by Gamerbolts** [comment]
No problem, i'll ask again in 3 months ;)

**Comment by Geforce132** [comment]
@Gamerbolts Hey, sorry, I just noticed this issue. I'll try to make sure that nothing in SecurityCraft is causing your crash. I'll create a pull request if I find the problem in LookingGlass' code. 

Also, sorry for the lack of updates to SecurityCraft, both bl4ckscor3 and I have been busy. We're planning on having one more update to the 1.7.10/1.8.x versions before moving to 1.11+ only. You can follow the development of SecurityCraft on Discord if ya' want: https://discord.gg/RYsKYx5

Lastly, I asked xCompWiz if I could possibly attempt to update LookingGlass to MC 1.11, and he said sure. So, a new port may possibly be coming, but I'm sure I'll have to ask xComp for pointers on certain bugs, haha.

**Comment by Gamerbolts** [comment]
Not a problem :) I hope you are able to figure out the problem if it is Securitycraft. And no problem for lack of updates. lol very understandable. Just went it works and I can have security cameras feed in my base for my modpack I will be very very happy lol. I'll join that discord as well. :) Thanks a lot

**Comment by Gamerbolts** [comment]
Still having this issue, no crashing however with the latest version of SC. bl4ckcor3 says it is an issue with looking glass, it can't handle a specific packet and fails.

**Comment by TheLastSpectre** [comment]
I have tested with the most recent version of Securitycraft (v.1.8.4), LookingGlass and NEIDs, and the issue is still there. So, none the updates to Securitycraft have solved the issue. That's not intended to be rude or anything, just stating what I know so as to help this get solved faster, because Securitycraft, LookingGlass, and Mystcraft are a staple in my modpack and I don't want to give up any of them.

**Comment by Gamerbolts** [comment]
Yep I have been waiting for a long time for a fix :( I've talked with everyone and still nothing. Really sucks because I would love this functionality in my modpack. But I gave up asking. As you can see over a year nothing more has been done

**Comment by TheLastSpectre** [comment]
Yeah, I know ;-;. I also learned that the issue definitely does NOT lie in Securitycraft, because the same issue occurs with NEIDs, Mystcraft and Lookingglass. So we can strike it off the list of culprits for the issue.

**Comment by Gamerbolts** [comment]
hmm that is interesting. Yeah my modpack is 340 mods so was hard to find the culprit lol i knew it was securitycraft, lookingglass and NEID. my bet is it's NEID and that mod author never replied to me :/

**Comment by Gamerbolts** [comment]
Just asked on the Securitycraft discord again, to see if I can stir up the conversation once more ...
